### PR TITLE
Exploring language semantics

### PR DESCRIPTION
**Pull request by lpil**
In this example I'm envisioning a language that is semantically very close to Ruby- Emerald is effectively another syntax for Ruby.

### CODE REVIEW

File: priv/classes.lisp [hunk 1]
```diff
@@ -0,0 +1,69 @@
+;
+; Exploring use of classes
+;
```
Comment (emilywoods) [inline priv/classes.lisp]: Nice examples! :)

File: priv/classes.lisp [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+(let ((fido (.new Dog))
+      (jane (.new Human)))
+
+  (Animal/legs)
+  (Dog/legs)
+  (Human/legs)
+
+  (.pay-taxes jane)
+  (.bark fido))
+
```
Comment (emilywoods) [inline priv/classes.lisp]: Is there an extra ')' here?

Comment (emilywoods) [inline priv/classes.lisp]: Oh no, maybe not - is it just the end of the class? All these parentheses are a little confusing! :P

Comment (lpil) [inline priv/classes.lisp]: It can get really easy to lose track of the closing parens I find.

Comment (lpil) [inline priv/classes.lisp]: It's the end of `let`

Comment (emilywoods) [inline priv/classes.lisp]: 👍

**Pull request merged**